src/test/java/teammates/test/driver/TestStringHelper.java

```diff
@@ -0,0 +1,23 @@
+package teammates.test.driver;
+
+import teammates.common.util.StringHelper;
+
+/**
+ * Holds additional methods for {@link StringHelper} used only in tests.
+ */
+public final class TestStringHelper {
```

Comment:
Option 2: I'd rather not test a class through a sub class because it's not clear what the SUT is.

Option 3: Something like `StringHelperTestHelper`?  It's a bit long but the suffix accurately describes the purpose of the class.